databroker/src/broker.rs

```diff
@@ -179,6 +180,23 @@ pub struct DataBroker {
     shutdown_trigger: broadcast::Sender<()>,
 }
 
+#[async_trait::async_trait]
+pub trait ActuationProvider {
+    async fn actuate(&self, actuation_changes: Vec<ActuationChange>) -> Result<(), tonic::Status>;
+}
+
+#[derive(Clone)]
+pub struct ActuationChange {
+    pub id: i32,
+    pub data_value: DataValue,
+}
+
+pub struct ActuationSubscription {
+    vss_ids: Vec<i32>,
+    actuation_provider: Box<dyn ActuationProvider + Send + Sync + 'static>,
+    permissions: Permissions,
```

Comment:
The permissions here are still unused: 
Where would be the best place to check the permissions? In val.rs / Provider / actuate (can't access them there though, since we only propagate the ActuationChanges and not the Subscriptions to val.rs)? The methods in broker itself already check permissions on call.

Reply:
Checked before calling "actuation_provider#actuate" (see broker.rs#actuate method)